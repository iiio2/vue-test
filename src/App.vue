<template>
  <div>
    <XArray v-model="myArray" />
    <pre>{{ myArray }}</pre>
  </div>
</template>

<script>
import XArray from "./components/XArray.vue";

export default {
  components: {
    XArray,
  },
  data: () => {
    return {
      myArray: ["one", "two"],
      myResult1: `<div>
        <input value="one">
        <input value="two">
        <input value="">
      </div>`,
    };
  },
};
</script>
