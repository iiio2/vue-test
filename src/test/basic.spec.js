import { mount } from "@vue/test-utils";
import App from "../App.vue";

describe("testing the app", () => {
  it("should mount", async () => {
    expect(App).toBeTruthy();

    const wrapper = mount(App);

    expect(wrapper.text()).toContain("one");
    expect(wrapper.html()).toMatchSnapshot();
  });
});
