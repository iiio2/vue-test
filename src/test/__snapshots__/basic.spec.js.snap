// Vitest Snapshot v1

exports[`testing the app > should mount 1`] = `
"<div>
  <div></div><pre>[
  \\"one\\",
  \\"two\\"
]</pre>
</div>"
`;
